wooey/tasks.py

```diff
@@ -183,3 +184,26 @@ def get_valid_file(cwd, name, ext):
     job.save()
 
     return (stdout, stderr)
+
+@celery_app.task(base=WooeyTask)
+def cleanup_wooey_jobs(**kwargs):
+    from datetime import timedelta, datetime
+    from .models import WooeyJob
+    # get the list of jobs that need deleting
+    cleanup_settings = wooey_settings.WOOEY_JOB_EXPIRATION
+    anon_settings = cleanup_settings.get('anonymous')
+    now = datetime.now()
```

Comment:
Good call. For some reason I thought Django was depracating their timezone in utils. Guess not!